includes/payment-methods/class-upe-payment-gateway.php

```diff
@@ -852,15 +868,13 @@ function() use ( $payment_fields ) {
 			?>
 
 			<fieldset id="wc-<?php echo esc_attr( $this->id ); ?>-upe-form" class="wc-upe-form wc-payment-form">
-				<div id="wcpay-upe-element"></div>
-				<div id="wcpay-upe-errors" role="alert"></div>
-				<input id="wcpay-payment-method-upe" type="hidden" name="wcpay-payment-method-upe" />
-				<input id="wcpay_selected_upe_payment_type" type="hidden" name="wcpay_selected_upe_payment_type" />
```

Comment:
Are these two inputs used or relied upon anywhere in the codebase? I couldn't find any reference to their names nor ids.